bitcointranscripts/transcription-review-front-end
# Admin page (Table Status, Search by username)

PR Checklist

- [x]  Data Display in a Table: 
- [x]  Filtering Options 
- [x] Default View: 

**How to Test**
To test this pr make sure you are an admin and navigate to `reviews/all` or use the sidebar navigation.
